src/v/cluster/cloud_metadata/manifest_downloads.cc

```diff
@@ -213,6 +214,9 @@ ss::future<cluster_manifest_result> download_highest_manifest_in_bucket(
               cluster_uuid_str);
             continue;
         }
+        if (ignore_uuid.has_value() && ignore_uuid.value() == cluster_uuid) {
```

Comment:
nit: optional has override for `opeartor==`: 
`ignore_uuid == cluster_uuid`

Reply:
Done